src/modules/feature-modules/accessor/pages/innovation/export/export-request.component.html

```diff
@@ -0,0 +1,81 @@
+<theme-content-wrapper [status]="pageStatus">
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <form [formGroup]="form">
+
+        <ng-container *ngIf="stepNumber === 1">
+
+          <p class="nhsuk-body-l">You need to request permission from the innovator to download and share their
```

Comment:
I believe that this looks better without "...body-l".

What do you think?

Reply:
I agree, changed in the last commit.